macro/shared/src/main/scala/monocle/macros/internal/Macro.scala

```diff
@@ -60,8 +60,14 @@ private[macros] class MacroImpl(val c: blackbox.Context) {
 
     val strFieldName = c.eval(c.Expr[String](c.untypecheck(fieldName.tree.duplicate)))
 
+    // when using 'private[package] val name' members, they get mangled to 'name$access$number' by the scala compiler
+    // so also accept these
+    def isNameOrMangledFrom(symbolName: String, fieldName: String): Boolean = {
```

Comment:
could you change the first parameter to be a `c.TermName` instead of `String` such as we call it `isNameOrMangledFrom(m.name, strFieldName)`?